src/lib/detection_algorithms/main.js

```diff
@@ -0,0 +1,50 @@
+//read from config to find detection algorthims and paths
```

Comment:
Consider placing this in a separate module (e.g., have a  "DetectionAlgorithmManger" folder and have this file inside) having a module that has more libraries of modules is uncommon practice in node.js..